test/__snapshots__/main.test.js.snap

```diff
@@ -0,0 +1,8 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`creates a hidden contenteditable div:  <div contenteditable="true" style="position: absolute; top:0; left: 0; opacity: 0;" /> 1`] = `
+<div
+  contenteditable="true"
+  style="position: absolute; top:0; left: 0; opacity: 0;"
+/>
+`;
```

Comment:
ooh what's this? do we use it?

Reply:
It's generated as a reference for comparing markup snippets without caring about the formatting (new lines or indentation). If an intended change is introduced the snapshot can be updates using the `--updateSnapshot` flag.
```
node_modules/jest/bin/jest.js --updateSnapshot
```